.travis.yml

```diff
@@ -1,11 +1,21 @@
 language: r
 sudo: required
 cache: packages
+dist: trusty
 
 apt_packages:
-  - r-cran-rgdal
   - libproj-dev
 
+before_install:
+  - sudo add-apt-repository ppa:ubuntugis/ubuntugis-unstable --yes
+  - sudo apt-get --yes --force-yes update -qq
+  - sudo apt-get install --yes libudunits2-dev libproj-dev libgeos-dev libgdal-dev
+
+r_packages:
```

Comment:
@hadley I thought you're talking about the sudo apt-get calls, especially the one in line 12. If you're not, just ignore my comment